style.css

```diff
@@ -237,6 +248,7 @@ footer {
 
 .footer-logo-image {
   align-items: center;
+  width: 10%;
```

Comment:
Do you need the logo to stretch?

Reply:
Yes, because if I remove this width:10% the icons become to large to fit in one line.